autoconfigure/stream-kafka/src/main/resource/META-INF/spring.factories

```diff
@@ -0,0 +1,2 @@
+org.springframework.boot.autoconfigure.EnableAutoConfiguration=\
+zipkin.autoconfigure.sparkstreaming.ZipkinSparkStreamingAutoConfiguration
```

Comment:
another typo (which made eyes cross)

the special directory for files is resources/ not resource/ :P